test/langtools/jdk/javadoc/doclet/testCopyFiles/TestCopyFiles.java

```diff
@@ -56,17 +56,17 @@ public void testDocFilesInModulePackages() {
                     "Hello World" (phi-WINDOW-TITLE-phi)""",
                 "phi-TOP-phi",
                 // check top navbar
-                "phi-HEADER-phi",
-                """
-                    <a href="../../module-summary.html">Module</a>""",
-                """
-                    <a href="../package-summary.html">Package</a>""",
                 """
-                    <a href="../../../overview-tree.html">Tree</a>""",
+                    <a href="../package-tree.html">Tree</a>""",
```

Comment:
What is the significance of this filename change?

Reply:
For context-specific links in the top navigation bar we always link to the target closest to the current page; for example, The "USE" link in class documentation links to the Class Use page of the current class, and the "TREE" link in files belonging to a package links to the Package Tree page. This was previously not handled correctly for package-level doc files. It is now fixed, so the test has to be updated.

Reply:
Thanks for the explanation.